api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirLocationDaoImplTest.java

```diff
@@ -47,20 +42,14 @@ public class FhirLocationDaoImplTest extends BaseModuleContextSensitiveTest {
 	
 	private static final String LOCATION_INITIAL_DATA_XML = "org/openmrs/module/fhir2/api/dao/impl/FhirLocationDaoImplTest_initial_data.xml";
 	
-	private FhirLocationDaoImpl fhirLocationDao;
-	
 	@Autowired
-	@Qualifier("sessionFactory")
-	private SessionFactory sessionFactory;
+	private FhirLocationDao fhirLocationDao;
 	
-	@Mock
-	private LocationService locationService;
+	@Autowired
+	LocationService locationService;
```

Comment:
Don't mock in context sensitive tests - just use the services as intended and set up any test data needed properly